packages/fxa-settings/src/components/DropDownAvatarMenu/index.tsx

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import Avatar from '../Avatar';
+import { useAccount } from '../../models';
+import { useClickOutsideEffect } from 'fxa-react/lib/hooks';
+import { useEscKeydownEffect } from '../../lib/hooks';
+import { ReactComponent as SignOut } from './sign-out.svg';
+
+export const DropDownAvatarMenu = () => {
+  const { displayName, primaryEmail } = useAccount();
+  const [isRevealed, setRevealed] = useState(false);
+  const toggleRevealed = () => setRevealed(!isRevealed);
+  const avatarMenuInsideRef = useClickOutsideEffect<HTMLDivElement>(
+    setRevealed
+  );
+  useEscKeydownEffect(setRevealed);
+  const dropDownId = 'drop-down-avatar-menu';
+
+  //TODO: mutation for Sign out
+
+  return (
+    <div className="relative" ref={avatarMenuInsideRef}>
+      <button
+        onClick={toggleRevealed}
+        data-testid="drop-down-avatar-menu-toggle"
+        title="Firefox Account Menu"
+        aria-expanded={isRevealed ? 'true' : 'false'}
+        aria-controls={dropDownId}
+        className="rounded-full border-2 border-transparent hover:border-purple-500 focus:border-purple-500 focus:outline-none active:border-purple-700 transition-standard"
+      >
+        <Avatar className="w-10 rounded-full" />
+      </button>
+      {isRevealed && (
+        <div
+          id={dropDownId}
+          data-testid={dropDownId}
+          className="drop-down-menu -left-52"
+        >
+          <div className="flex flex-wrap">
+            <div className="flex w-full p-4">
+              <div className="mr-3 flex flex-1">
+                <Avatar className="w-10" />
+              </div>
+              <div className="flex flex-4 truncate">
+                <p className="leading-5 max-w-full">
+                  <span className="text-grey-400 text-xs">Signed in as</span>
```

Comment:
just curious, do we have a localization / RTL plan for this component / globally? IIRC `fluent-react` requires some additional work for inserting variables inside strings, but it's been a while.

Reply:
I have #5169 sitting in an l10n user story in which I'm hoping we can use that tailwindcss package linked in there for, but it might end up being a much larger task if we can't.